### ISSUE TITLE
Conisder navigation of report

### ISSUE THREAD

**Issue by harrylandles** [JohnAdders/climate_narrative#91]
From Leo Donnachie,
Presentation currently makes the reports a little difficult to navigate – will there be any tweaks to make it more readable for the user?

**Comment by JohnAdders** [comment]
Agreed, we are looking into some this, some short term improvements and longer term ones
